client/js/components/statement/listStatements/ExportModal.vue

```diff
@@ -0,0 +1,241 @@
+            :placeholder="Translator.trans(column.placeholder)"
+            :width="column.width"
+            v-model="column.title"
+            @input="handleInput" />
+        </div>
+      </section>
+
+      <div class="flex justify-end">
+        <dp-button-row
+          primary
+          secondary
+          :primary-text="Translator.trans('export.statements')"
+          :secondary-text="Translator.trans('abort')"
+          @primary-action="handleExport"
+          @secondary-action="handleAbort" />
+      </div>
+    </dp-modal>
+  </div>
+</template>
+
+<script>
+import {
+  DpButtonRow,
+  DpContextualHelp,
+  DpInput,
+  DpModal,
+  DpRadio,
+  sessionStorageMixin
+} from '@demos-europe/demosplan-ui'
+
+export default {
+  name: 'ExportModal',
+
+  components: {
+    DpButtonRow,
+    DpContextualHelp,
+    DpInput,
+    DpModal,
+    DpRadio
+  },
+
+  mixins: [sessionStorageMixin],
+
+  data () {
+    return {
+      active: 'docx',
+      docxColumns: {
+        left: {
+          width: 'col-span-1',
+          dataCy: 'exportModal:input:colLeft',
+          placeholder: Translator.trans('segments.export.segment.id'),
+          title: null
+        },
+        middle: {
+          width: 'col-span-2',
+          dataCy: 'exportModal:input:colMiddle',
+          placeholder: Translator.trans('segments.export.statement.label'),
+          title: null
+        },
+        right: {
+          width: 'col-span-2',
+          dataCy: 'exportModal:input:colRight',
+          placeholder: Translator.trans('segment.recommendation'),
+          title: null
+        }
+      },
+      exportTypes: {
+        docx: {
+          label: 'export.statements.docx',
+          uploadPath: 'dplan_statement_segments_export',
+          dataCy: 'exportModal:export:docx'
+        },
+        zip: {
+          label: 'export.statements.zip',
+          uploadPath: 'dplan_statement_segments_export_packaged',
+          dataCy: 'exportModal:export:zip'
+        },
+        xlsx: {
+          label: 'export.statements.xlsx',
+          uploadPath: 'dplan_statement_xls_export',
+          dataCy: 'exportModal:export:xlsx'
+        }
+      }
+    }
+  },
+
+  methods: {
+    applyDefaultPlaceholdersForEmptyNames () {
+      Object.keys(this.docxColumns).forEach(key => {
+        if (!this.docxColumns[key].title) {
+          this.docxColumns[key].title = null
+          this.docxColumns[key].placeholder = this.getDefaultPlaceholderByKey(key)
+        }
+      })
+    },
+
+    closeModalAndResetValues () {
+      this.$refs.exportModalInner.toggle()
+      this.active = 'docx'
+      this.resetColumnsUnsavedValues()
+    },
+
+    getDefaultPlaceholderByKey (key) {
+      const defaultPlaceholders = {
+        left: Translator.trans('segments.export.segment.id'),
+        middle: Translator.trans('segments.export.statement.label'),
+        right: Translator.trans('segment.recommendation')
+      }
+
+      return defaultPlaceholders[key]
+    },
```

Comment:
I looks like you do something weird with the placeholders.
Why do You have to reset them?

Reply:
because the placeholders could be different depending on what is saved in sessionStorage. I mean, if there is nothing saved in sessionStorage, the default placeholders are used. If there are saved values in sessionStorage, they will be used as the placeholders...the user needs to know/see the titles of the table headers are at the moment when the window is opened

Reply:
Yes. but if there is something stored, that values should not be displayed as placeholders, but as value from that input.

Reply:
The placeholder it self should not change